libraries/skeletons.md

```diff
@@ -6,6 +6,25 @@ title: Project Skeletons
 These application skeletons get you started in no time!
 
 
+### [Play application with Scala.js](https://github.com/vmunier/play-with-scalajs-example)
+
+Example application showing how you can integrate [Play](https://www.playframework.com/) with Scala.js.
+
+### [AWS Lambda with Scala.js](https://github.com/bgahagan/scalajs-lambda.g8)
+
+A skeleton to build an AWS Lambda handler using Scala.js 1.0. Uses sbt-scalajs-bundler to manage npm dependencies and sbt-native-packager to package lambda for deployment.
+
+### [npm scalajs-starter](https://github.com/konradst/scalajs-starter)
+
+Scala.js Node.js skeleton app / library handled with ```npm start```, ```build```, ```test```, ```clean``` commands.
```

Comment:
This one seems outdated. It uses sbt-scalajs 1.0.0-M8.

Reply:
Indeed, I was keeping everything related to 1.0.0 as recent, I'll update the section